apps/todo-app/src/app/components/todolist.tsx

```diff
@@ -29,6 +29,15 @@ export const Todos = () => {
           body: JSON.stringify({ task: newTask })
         });
 
+        if (!res.ok)
+          {if (res.status === 401) {
+            // Redirect user back to the sign-in page
+            window.location.href = '/';
```

Comment:
recommend using React instead of Window API?

Reply:
suggested using useNavigate